### PR TITLE
Goodbye Python 2.7,  hello Python 3.6!

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,44 +1,22 @@
 language: python
+
 python:
   - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.9"
```
Comment (bernt-matthias) [inline .travis.yml:7]: Maybe skip 3.9 for now, seems not yet compatible with Galaxy -- or the other way round :)

**Comment by bernt-matthias** [comment]
Do you think its a good idea to add those two python scripts: 

- https://github.com/galaxyproteomics/tools-galaxyp/blob/bb993bb141e6a2158be00c4335a01d0f0dca1dfc/tools/openms/fill_ctd.py
    - its needed for executing the Galaxy tools (it takes the json generated by Galaxy and fills it into a param file)
- https://github.com/galaxyproteomics/tools-galaxyp/blob/bb993bb141e6a2158be00c4335a01d0f0dca1dfc/tools/openms/fill_ctd_clargs.py
    - It takes a command line for a tool and fills the values given on the command line into a param file

The first is already in the test-data dir .. and its quite an extra effort for me to keep them in sync.

**Comment by mr-c** [comment]
> Do you think its a good idea to add those two python scripts:

I don't follow, sorry. Is this associated with this PR? Or another issue?

**Comment by bernt-matthias** [comment]
> Is this associated with this PR? Or another issue?

Just a general question...

**Comment by mr-c** [comment]
> its quite an extra effort for me to keep them in sync.

Maybe ask for them to be moved to a general purpose python package that CTDConverter could require for testing?

Like https://pypi.org/project/galaxy-util/ ?

**Pull request merged**